mocko-proxy/test/proxy.js

```diff
@@ -25,4 +25,9 @@ module.exports = (mocko, _describe, it) => () => {
         const { status } = await mocko.post('/validate/query?foo=bar');
         expect(status).to.equal(201);
     });
+
+    it('should ignore files and directories starting with dot(.)', async () => {
+        const { status } = await mocko.post('/dot-route');
+        expect(status).to.equal(404);
```

Comment:
As the request returns a 404 code, the mocko.post will reject. You might need something like:
```js
let code;

await mocko.get('/dot-route')
    .then(({ status }) => code = status)
    .catch(({ response }) => code = response.status);

expect(code).to.equal(404);
```

Reply:
I didn't go into the documentation for mocko.post(), thanks for pointing that out.
I will change it right away.